x-pack/plugins/security_solution/public/timelines/components/timeline/data_providers/helpers.tsx

```diff
@@ -342,3 +342,18 @@ export const addContentToTimeline = ({
     });
   }
 };
+
+export const convertIsOneOfQueryStringArrayToDisplayValue = (
```

Comment:
The display value could be more precise than the current implementation. However since I don't think it needs to be, and this implementation is less expensive than a similar regex one, I opted for it.